### PR TITLE
update nightly

### PR DESCRIPTION
**Pull request by Hoverbear**
Update to latest nightly. :)

### CODE REVIEW

File: src/pd/timestamp.rs [hunk 1]
```diff
@@ -125,7 +125,7 @@ struct TsoRequestStream<'a> {
 }
 
 impl<'a> Stream for TsoRequestStream<'a> {
-    type Item = (TsoRequest, WriteFlags);
+    type Item = Result<(TsoRequest, WriteFlags)>;
```
Comment (nrc) [inline src/pd/timestamp.rs]: Why do we need this? Seems like we'll never use the `Err` case

Comment (Hoverbear) [reply src/pd/timestamp.rs]: It's needed for `join!` to work since it wants a `TryStream`.

Comment (nrc) [reply src/pd/timestamp.rs]: I'm not sure why `join` needs a `TryStream`, it should work fine with a regular stream. Do we use `join` with a different `TryStream` and we need this one to match? I'm not sure if it makes it any better, but you could use `.map(Ok)` to turn the stream into one which implements `TryStream`.

Comment (Hoverbear) [reply src/pd/timestamp.rs]: Should be fixed :)

**Pull request merged**